desc/plotting.py

```diff
@@ -4000,6 +4009,285 @@ def plot_field_lines(
+    **kwargs : dict, optional
+        Specify properties of the figure, axis, and plot appearance e.g.::
+
+            plot_X(figsize=(4,6),)
+
+        Valid keyword arguments are:
+
+        * ``color``: color to use for field lines, default is black.
+        * ``figsize``: tuple of length 2, the size of the figure in inches
+        * ``lw``: float, linewidth of plotted field lines
+        * ``ls``: str, linestyle of plotted field lines
+        * ``showgrid``: Bool, whether or not to show the coordinate grid lines.
+          True by default.
+        * ``showticklabels``: Bool, whether or not to show the coordinate tick labels.
+          True by default.
+        * ``showaxislabels``: Bool, whether or not to show the coordinate axis labels.
+          True by default.
+        * ``zeroline``: Bool, whether or not to show the zero coordinate axis lines.
+          True by default.
+
+        Additionally, any other keyword arguments will be passed on to
+        ``desc.particles.trace_particles``
+
+    Returns
+    -------
+    fig : plotly.graph_objs._figure.Figure
+        Figure being plotted to.
+    plot_data : dict
+        Dictionary of the data plotted, only returned if ``return_data=True``
+        Contains keys ``["X","Y","Z","R","phi"]``, each entry in the dict is a list
+        of length `R0.size` corresponding to the number of field lines, and each
+        element of that list is an array of size `phis.size` corresponding to the
+        coordinate values along that field line.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        import desc
+        from desc.plotting import plot_particle_trajectories
+        from desc.particles import (
+            ManualParticleInitializerLab,
+            VacuumGuidingCenterTrajectory,
+        )
+
+        # One can either use the Equilibrum or MagneticField object
+        # to plot the particle trajectories. Here we will use eq to
+        # get the initial position and plotting the LCFS. Particle tracing will
+        # use the field object.
+        eq = desc.examples.get("precise_QA")
+        field = desc.io.load("../tests/inputs/precise_QA_helical_coils.h5")
+
+        rhos = [0.5]
+
+        fig = plot_3d(eq, "|B|", alpha=0.5)
+        # For demo purposes, we show how to initialize particles
+        # in the lab frame using the flux coordinates as inputs.
+        # Initializer will perform the conversion from flux to lab coordinates
+        # using the equilibrium object.
+
+        # Note: Initializers return the cordinates in the frame specified by
+        # the trajectory model.
+        initializer = ManualParticleInitializerFlux(
+            rho0=rhos,
+            theta0=0,
+            zeta0=0,
+            xi0=0.7,
+            E = 1e-1,
+            m = 4.0,
+            q = 1.0,
+            eq=eq, # needed for flux->lab conversion
+        )
+        # One can also use frame='flux', and pass eq to
```

Comment:
this is a bit confusing comment, perhaps adding a commented out line showing the alternative could be useful